src/components/population-space.tsx

```diff
@@ -0,0 +1,60 @@
+import { inject, observer } from "mobx-react";
+import * as React from "react";
+import { BaseComponent, IBaseProps } from "./base";
+
+import "./population-space.sass";
+
+interface IProps extends IBaseProps {}
+interface IState {}
+
+@inject("stores")
+@observer
+export class PopulationSpaceComponent extends BaseComponent<IProps, IState> {
+
+  public render() {
+    return (
+      <div className="population-space">
+        {this.RenderLeftPanel()}
+        {this.RenderRightPanel()}
+      </div>
+    );
+  }
+
+  private RenderLeftPanel() {
+    return (
+      <div className="population-panel left-abutment">
+        <div className="header">
+          <div className="title">Investigate: Population</div>
+        </div>
+        <div className="content">content here</div>
+      </div>
+    );
+  }
+
+  private RenderRightPanel() {
+    const {ui} = this.stores;
```

Comment:
Agreed about eventually refactoring state - we may want to create a separate UI store for each space

Reply:
Agree.  We'll need a story for that.